pdesolvers/heat_equation_1d/solver/cn_solver.py

```diff
@@ -65,4 +66,26 @@ def __build_tridiagonal_matrix(a, b, c, nodes):
 
         matrix = csc_matrix(matrix)
 
-        return matrix
\ No newline at end of file
+        return matrix
+
+    def plot(self):
```

Comment:
i feel like this plot function could be moved elsewhere as it is a shared function (any suggestions?)

Reply:
You plot the solution, not the solver, so it can be moved to a `Solution` class. Then every solver can return an instance of `Solution` that can be plotted.